megamek/src/megamek/common/Compute.java

```diff
@@ -254,6 +254,20 @@ public static float randomFloat() {
         return random.randomFloat();
     }
 
+    /**
+     * Selects a random element from a list
+     * @param list The list of items to select from
+     * @return     An element in the list
+     * @param <T>  The list type
+     */
+    public static<T> T randomListElement(List<T> list) {
```

Comment:
I was sure we had that somewhere but couldnt find it. But it might be placed in CollectionUtil

Reply:
I was pretty sure it was somewhere but I couldn't find it either.